### PR TITLE
WIP staged libsbp integration WIP

### PR DESCRIPTION
**Pull request by mfine**
DO NOT MERGE

This builds on the changes in #278. It removes `sbp_piksi.[ch]` files and integrates the firmware with the messages from libsbp. It required a couple of minor changes to libsbp captured in [mfine-integrate-libsbp-stage](https://github.com/swift-nav/libsbp/compare/mfine-integrate-libsbp-stage). This should realize full integration with libsbp. Building the PR for review and running in the test beds.

/cc @mookerji @fnoble

DO NOT MERGE

### CODE REVIEW

File: src/sbp_piksi.h [hunk 1]
```diff
@@ -1,177 +0,0 @@
...
-
-#define MSG_BASE_POS                0x44
-typedef struct __attribute__((packed)) {
-  double pos_llh[3];
-} msg_base_pos_t;
-
-typedef struct __attribute__((packed)) {
-  u32 P;     /**< Pseudorange (cm) */
-  struct __attribute__((packed)) carrier {
-    s32 Li;  /**< Carrier phase (integer seconds) */
```
Comment (mfine) [inline src/sbp_piksi.h]: @mookerji @fnoble s32 here, u32 in `carrier_phase_t` in `libsbp/lib.h`

Comment (fnoble) [inline src/sbp_piksi.h]: oops, it really does need to be signed!

Comment (mookerji) [inline src/sbp_piksi.h]: fascinating.

File: src/sbp_utils.c [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+#include "sbp_utils.h"
+
+/** \addtogroup sbp
+ * \{ */
+
+/** \defgroup sbp_utils SBP Utils
+ * Convert to and from SBP message types and other useful functions.
+ * \{ */
+
+void sbp_make_gps_time(msg_gps_time_t *t_out, gps_time_t *t_in, u8 flags)
```
Comment (mookerji) [inline src/sbp_utils.c]: Double check.

**Pull request merged**